<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>990</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1C2833;
color: white;
font-size: 18px;
font-family: &quot;Verdana&quot;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="sourceCodeGroupBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212F3D;</string>
    </property>
    <property name="title">
     <string>Cod sursă</string>
    </property>
    <widget class="QPlainTextEdit" name="codeTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>138</y>
       <width>631</width>
       <height>351</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#1C2833;
</string>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="tabStopWidth">
      <number>80</number>
     </property>
    </widget>
    <widget class="QPushButton" name="importFileBtn">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>27</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #273746;</string>
     </property>
     <property name="text">
      <string>Importă soluția</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Importă un fișier cu rezolvarea:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>67</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>sau</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rezolvă problema în zona de cod mai jos:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generateFeedbackBtn">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>500</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #273746;</string>
     </property>
     <property name="text">
      <string>Rulează</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>501</y>
       <width>341</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="saveCodeBtn">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>500</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #273746;</string>
     </property>
     <property name="text">
      <string>Salvează</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="feedbackGroupBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>651</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212F3D;</string>
    </property>
    <property name="title">
     <string>Feedback</string>
    </property>
    <widget class="QTabWidget" name="feedbackTabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1C2833;

</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="consola">
      <attribute name="title">
       <string>Consolă</string>
      </attribute>
      <widget class="QLabel" name="feedbackResultLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>501</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Soluția ta nu a trecut toate testele. Mai încearcă.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="feedbackResultIcon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="feedback">
      <attribute name="title">
       <string>Sugestii</string>
      </attribute>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>611</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#212F3D;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="hintLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>611</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #B562FF;</string>
        </property>
        <property name="text">
         <string>Codul tău nu funcționează dacă...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="wizardIcon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="hintBtn">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>18</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #273746;</string>
       </property>
       <property name="text">
        <string>Dă-mi un pont</string>
       </property>
      </widget>
      <widget class="QLabel" name="wizardHelpLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>23</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ai tot încercat și te-ai blocat?</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="algoritm">
      <attribute name="title">
       <string>Algoritm</string>
      </attribute>
      <widget class="QLabel" name="crystalHelpLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>23</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Nu știi de unde să începi?</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="crystalBallIcon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="solvingHintBtn">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>18</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #273746;</string>
       </property>
       <property name="text">
        <string>Afișează rezolvarea</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>611</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#212F3D;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="solvingHintLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>611</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>O metodă simplă de rezolvare a problemei ar fi...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="projectGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>651</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #212F3D;</string>
    </property>
    <property name="title">
     <string>Proiect</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>30</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Alege o problemă pe care vrei să-l rezolvi:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="projectsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#1C2833;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="projectNameLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>9</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Denumire proiect</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>211</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Date de intrare</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>274</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Date de ieșire</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectRequirementLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>64</y>
        <width>611</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="projectInputLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>218</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #212F3D;</string>
      </property>
      <property name="text">
       <string>&lt;span style='color:#FF8A00'&gt;a&lt;/span&gt; = 12&lt;br&gt;&lt;span style='color:#FF8A00'&gt;b&lt;/span&gt; = 8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="projectOutputLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>281</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #212F3D;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>182</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Exemplu:</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectDifficultyLabel">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>5</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#1C2833;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>139</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Signatura funcției:</string>
      </property>
     </widget>
     <widget class="QLabel" name="functionSignatureLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>139</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>def sumaCifrelor(a)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="testGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>651</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#212F3D;</string>
    </property>
    <property name="title">
     <string>Rezultat</string>
    </property>
    <widget class="QTabWidget" name="testTabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #1C2833;

</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="teste">
      <attribute name="title">
       <string>Teste</string>
      </attribute>
      <widget class="QLabel" name="totalNoTests">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>90</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Nr de teste rulate: 27</string>
       </property>
      </widget>
      <widget class="QLabel" name="noPassedTests">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
color: #18CB3B;</string>
       </property>
       <property name="text">
        <string>PASSED: 22</string>
       </property>
      </widget>
      <widget class="QLabel" name="noFailedTests">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
color: #E70026;</string>
       </property>
       <property name="text">
        <string>FAILED:  4</string>
       </property>
      </widget>
      <widget class="QLabel" name="testsResult">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>501</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Soluția ta NU a trecut toate testele.</string>
       </property>
      </widget>
      <widget class="QLabel" name="testsResultIcon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="rezumatTeste">
      <attribute name="title">
       <string>Rezumat teste</string>
      </attribute>
      <widget class="QTableWidget" name="testsTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>611</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
